src/_zkapauthorizer/sql.py

```diff
@@ -27,7 +110,7 @@ class StorageAffinity(Enum):
     NUMERIC = auto()
 
 
-@define(frozen=True)
+@frozen
```

Comment:
incidental, I was trying to fix a problem where hypothesis suddenly became unable to automatically build Columns.  this change didn't help but it seemed worth keeping.